provider/lxd/environ.go

```diff
@@ -230,18 +248,19 @@ func (env *environ) AvailabilityZones(ctx context.ProviderCallContext) ([]common
 	// If we are not using a clustered server (which includes those not
 	// supporting the clustering API) just represent the single server as the
 	// only availability zone.
-	if !env.server.IsClustered() {
+	server := env.server()
+	if !env.server().IsClustered() {
```

Comment:
You created `server` the line above. Perhaps use that?

Reply:
i meant to, sigh